support_great_writers.php

```diff
@@ -5,7 +5,7 @@
 Description: Sell Amazon products in sidebar widgets, unique to the individual POST or generically from a default pool of products that you define.
 Author: HeyPublisher
 Author URI: https://www.heypublisher.com
-Version: 3.1.0
+Version: 3.1.1
```

Comment:
This needs to be fixed in `readme.txt` too, and `CHANGELOG.md` should be updated with changes in this release.